war/package.json

```diff
@@ -3,6 +3,8 @@
   "version": "1.0.0",
   "description": "Jenkins User Interface",
   "license": "MIT",
+  "author": "Tom Fennelly <tom.fennelly@gmail.com> (https://github.com/tfennelly)",
```

Comment:
I can remove this line, it is unnecessary. I used it because @tfennelly designed everything around `js-modules`.